---
sidebar: id-links
---

## Current Research
<figure style="float: right; max-width: 300px; margin-left: 15px; margin-bottom: 10px; text-align: center;">
  <a href="/assets/images/plasticell_poster.jpg" target="_blank">
    <img src="/assets/images/plasticell_poster.jpg" alt="Plasticell poster" style="width: 100%;" />
  </a>
  <figcaption style="text-align: left; font-size: 0.9em; color: #555;">Presenting our NK cell research</figcaption>
</figure>

I'm currently a data scientist at Cell and Gene Therapy Catapult, working on projects to help scale-up and improve the production of cell and gene therapies with bioinformatics, modelling, and other tools.

We recently completed a project to scale-up the production of NK cells with the company Plasticell. I performed integrated multi-omics analysis on the data from this project to identify promising biomarkers, which will be available shortly as a pre-print/paper.

</br>
</br>

## Postdoc Research
<figure style="float: right; max-width: 300px; margin-left: 15px; margin-bottom: 10px; text-align: center;">
  <a href="/assets/images/plastid_preview.jpg" target="_blank">
    <img src="/assets/images/plastid_preview.jpg" alt="Plastid Preview 2023" style="width: 100%;" />
  </a>
  <figcaption style="text-align: left; font-size: 0.9em; color: #555;">Plastid Preview 2023 at New College</figcaption>
</figure>

As a postdoctoral researcher at the University of Oxford, I led on a project editing the SP1 and SP2 genes in wheat (components of the chloroplast-associated protein degradation system) to develop new varieties of stay-green wheat (a phenotype associated with improved stress tolerance). An overview of this work is shown in a [poster I presented](https://doi.org/10.5281/zenodo.10262994) at Plastid Preview 2023.

The lines I developed are now being grown in multiple field trials across the UK to assess their performance, see [this link](https://www.isaaa.org/kc/cropbiotechupdate/article/default.asp?ID=21200) for more information.

</br>

## PhD Research
<figure style="float: right; max-width: 300px; margin-left: 15px; margin-bottom: 10px; text-align: center;">
  <a href="/assets/images/DSC_7428.jpg" target="_blank">
    <img src="/assets/images/DSC_7428.jpg" alt="Rothamsted team" style="width: 100%;" />
  </a>
  <figcaption style="text-align: left; font-size: 0.9em; color: #555;">The Rothamsted team in the gene-edited trial</figcaption>
</figure>

I completed my PhD at Rothamsted Research with Nigel Halford (along with partners at the University of Reading and Mondelez International) on the genetic and agronomic control of free asparagine accumulation in wheat. The projects I carried out are described in more detail below. 

I also helped my team undertake analysis of the first field trial of gene-edited wheat in Europe. The scientific article can be read [here](https://onlinelibrary.wiley.com/doi/10.1111/pbi.14026?msockid=2d7c6711fd7364ff025a72acfcf6655d) and some of the media coverage can be seen [here](https://www.dailymail.co.uk/sciencetech/article-9950917/British-scientists-growing-new-strain-gene-edited-wheat.html).

### Investigating natural diversity in the asparagine synthetase family

In collaboration with a group at Colorado State University, we investigated a deletion of one of the wheat asparagine synthetase genes (*TaASN-B2*) and how this impacts grain asparagine content in the field. Our paper on this is available [here](https://bmcplantbiol.biomedcentral.com/articles/10.1186/s12870-021-03058-7) and code/data is available [here](https://zenodo.org/record/7760909#.ZCMNeXbMJaQ).

### Finding stable QTL for quality traits and amino acids

Using a soft wheat mapping population developed in collaboration with John Innes Centre and Limagrain, we investigated stable QTL for asparagine and other quality traits that can be incorporated into breeding programs. I also looked at the capacity for genomic prediction to be used for these traits and leveraged recently released wheat pangenome resources to identify underlying candidate genes controlling lysine content. The article from this work is available [here](https://acsess.onlinelibrary.wiley.com/doi/10.1002/tpg2.20335). The data is available at Dryad [here](https://doi.org/10.5061/dryad.b8gtht7hj) and the code is available at Zenodo [here](https://zenodo.org/record/7759333).

### Growing low asparagine wheat
<figure style="float: right; max-width: 300px; margin-left: 15px; margin-bottom: 10px; text-align: center;">
  <a href="/assets/images/JAFC_cover.jpg" target="_blank">
    <img src="/assets/images/JAFC_cover.jpg" alt="JAFC cover" style="width: 100%;" />
  </a>
  <figcaption style="text-align: left; font-size: 0.9em; color: #555;">Our JAFC cover</figcaption>
</figure>

In combination with the above genetic studies, I also investigated the agronomic control (with varying rates of nitrogen, sulphur, potassium, and phosphorus fertiliser) of free asparagine in two field trials. In parallel, I looked into whether multispectral measurements of plants and grain could be used to predict asparagine content. The impact of these factors on end products was also assessed by baking biscuits (since we were investigating soft wheat) at the Mondelez laboratories in Reading and analysing their acrylamide content.

The article from this work is available [here](https://pubs.acs.org/doi/full/10.1021/acs.jafc.2c07208) and code/data is available [here](https://zenodo.org/record/7760903#.ZCMNXHbMJaQ).

Our photograph of biscuits with varying acrylamide content (ranging from low to high from light to darker colour) was selected for the cover of this issue of JAFC.

### Neural networks for prediction of grain asparagine content

I re-analysed data from my PhD (see below) to assess the accuracy of neural networks for prediction of grain asparagine content from imaging data. The results of this analysis are available on bioRxiv [here](https://doi.org/10.1101/2023.12.04.569839).
